backend/graphql/service/user.service.ts

```diff
@@ -15,4 +23,34 @@ export class UserService {
     }
     return user
   }
+
+  async login(input: LoginInput, context: Context) {
+    const errorMessage = 'Invalid email or password'
+
+    const user = await UserModel.findOne({ email: input.email }).lean()
+
+    if (!user) {
+      throw new NotFoundError(errorMessage)
+    }
+
+    const isValidPassword = await bcrypt.compare(input.password, user.password)
+
+    if (!isValidPassword) {
+      throw new NotFoundError(errorMessage)
+    }
+
+    const { _id } = user
+
+    const token = generateToken(_id)
+
+    context.res.cookie('accessToken', token, {
+      maxAge: 1000 * 60 * 60, // 1 hour
+      httpOnly: true,
+      domain: 'localhost',
```

Comment:
What happens when you deploy the application?

the domain is not gonna be localhost